sky/serve/serve_state.py

```diff
@@ -43,14 +44,22 @@ def create_table(cursor: 'sqlite3.Cursor', conn: 'sqlite3.Connection') -> None:
         replica_id INTEGER,
         replica_info BLOB,
         PRIMARY KEY (service_name, replica_id))""")
-
+    cursor.execute("""\
+        CREATE TABLE IF NOT EXISTS versions (
```

Comment:
Seems this table is mainly for storing the `spec`, should we call it `service_specs`?

Reply:
Just curious, do we need to store the specs for the older version? Should we just store the latest spec in the `services` table instead?

Reply:
It is possible that a service can have more than 2 concurrent versions. We kinda want to keep track of all these old versions. 
I think the main purpose of the table is to keep track of different versions; `service_specs` doesn't seem to capture that

Reply:
I see, that makes sense. How about we name it `version_specs`? It seems we are using that table as a source of specs for different version as well. Would like to make it more clear.